plugins/modules/zos_job_submit.py

```diff
@@ -89,8 +89,8 @@
     description:
       - Specifies which encoding the local JCL file should be converted from
         and to, before submitting the job.
-      - If this parameter is not provided, and the z/OS systems default encoding can not be identified,
-        the JCL file will be converted from ISO8859-1 to IBM-1047 by default.
+      - If this parameter is not provided, the JCL file will be converted from
```

Comment:
@richp405 this is also the same issue, dates back to deliberate doc changes for the better are being reversed by your commit. I will halt this review and as you to rebase so that I am not questioning every change.